embeddings/metric/unit_seqeval_metric.py

```diff
@@ -6,7 +6,7 @@
 from embeddings.metric.hugging_face_metric import HuggingFaceMetric
 
 
-class POSTaggingSeqevalMetric(HuggingFaceMetric):
+class UnitSeqevalMetric(HuggingFaceMetric):
```

Comment:
`U-` is from `unit`? :) I didn't know it.

Reply:
Yes, in BILOU, U- is from unit